pages/index.js

```diff
@@ -151,12 +144,28 @@ export default function Home() {
           </div>
 
           {/* Right column */}
-          <Image
-            src="https://res.cloudinary.com/delete-44/image/upload/l_text:v1644177732:Inscryption:HEAVYWEIGHT.ttf_96:NAMEHERE,g_north,y_180/c_scale,l_v1644060029:Inscryption:stinky.svg,w_248,y_340/v1644060066/Inscryption/blank_card.webp"
-            alt="A blank card with the 'Stinky' sigil"
-            width={640}
-            height={1048}
-          />
+          <div className="width-full flex justify-center items-center">
+            {busy ? (
+              <div
+                className="spinner-border animate-spin inline-block w-8 h-8 border-4 border-orange-400 rounded-full"
+                role="status"
+              >
+                <span className="visually-hidden">Loading...</span>
+              </div>
+            ) : (
+              <></>
+            )}
+
+            <Image
+              src={`https://res.cloudinary.com/delete-44/image/upload/${nameTF}l_v1644060029:Inscryption:stinky.svg,w_248,y_340/v1644060066/Inscryption/blank_card.webp`}
+              alt="A blank card with the 'Stinky' sigil"
```

Comment:
you're a stinky sigil

Reply:
Thanks for the feedback! We love hearing each and every opinion here at Inscryber Ltd, and yours most of all. We wil take this on board and make adjustments accordingly!